.github/workflows/lint-check.yml

```diff
@@ -17,4 +17,8 @@ jobs:
 
       - name: Lint
         # `pnpm lint` is not emit error exit code, so we need to check the output
-        run: pnpm lint | tee >(tail -n1 | grep -q 'Failed') && exit 1 || exit 0
+        run: |
+          OUTPUT=$(pnpm lint)
+          echo "$OUTPUT"
+          LAST_LINE=$(echo "$OUTPUT" | tail -n1)
+          echo "$LAST_LINE" | grep -q 'Failed' && exit 1 || exit 0
```

Comment:
[commet]
I didn't recognise it, but -no-bail doesn't cause an error!
This is not good.
You can hack it with CI, but in this case replace it with npm-run-all2. That way we can get the behaviour we want locally.
I'll send a PR later

Reply:
yes this is not good solution (https://github.com/wrtnlabs/agentica/pull/205#discussion_r2009535053)
Thank you for suggesting a better way.
If you don't upload PR, I'll check it later and work on it.

Reply:
I'll issue it
If so, development properties can link issues and requests

Reply:
I would like to thank you for your hack! Thank you.